events/activities/hold_court_activity/hold_court_events_general.txt

```diff
@@ -37234,10 +37234,10 @@ hold_court.8190 = {
 scripted_trigger hold_court_8200_infidel_trigger = {
 	is_available_ai_adult = yes
 	NOT = { faith = root.faith }
-	NOR = {
-		# Warcraft
+	# Warcraft
+	NOT = {
 		save_temporary_scope_as = infidel
```

Comment:
I would move it to the top of the trigger. It's not a mistake but code style remark.
Resolve once you do.